Client/Frontend/Theme/ThemedWidgets.swift

```diff
@@ -4,18 +4,28 @@
 import UIKit
 
 class ThemedTableViewCell: UITableViewCell, Themeable {
+
+    var detailTextColor: UIColor = UIColor.theme.tableView.disabledRowText
+
     override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
-        applyTheme()
     }
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
     }
 
+    override func willMove(toWindow newWindow: UIWindow?) {
+        super.willMove(toWindow: newWindow)
+        if newWindow != nil {
+            // UIView appearing
+            applyTheme()
```

Comment:
Previously, applyTheme was happening in init, but the properties of this will be set *after* init but before viewWillAppear. This is a UIView equivalent to viewWillAppear.